content/denver-ptg-2018.rst

```diff
@@ -0,0 +1,340 @@
+:title: OpenStack Project Team Gathering: Denver round 2
+:slug: denver-ptg-2
+:sortorder: 30
+:date: 2018-09-21 17:00
+
+In our return to the Renaissance Denver Hotel, I participated mainly in keystone
+and TC tracks.
+
+In keystone land, we spent time on our recurring themes from past cycles, namely
+RBAC improvements and unified limits. We focused less on application
+credentials, as the plans from last cycle still apply for this cycle. We
+revamped plans for federation improvements, given the increasing interest from
+the Edge computing groups and feedback from the user survey. We also solidified
+plans for implementing JWT.
+
```

Comment:
I suppose there is a lot of tribal knowledge for the average reader: what is TC, what is JWT?

If your audience is openstack developers or only your manager, that might be good, but an extra explanation/link doesn't hurt, IMO.